views/submit-payment.ejs

```diff
@@ -35,9 +35,8 @@
         <form action="<%= next_url %>">
 
             <div class="form-group">
-
                 <button type="submit" class="govuk-button" data-module="govuk-button">Pay</button>
-
+                <div data-testid="app-id" data-value="<%= applicationId %>" />
```

Comment:
since it's a testid you can just stick it on the button or form instead of a new element 👍

Reply:
ah yes that's a good point, I will do that